pivotal.py

```diff
@@ -39,9 +39,12 @@ def parse(self, response):
 
             if len(reference) > 0 and len(url) > 0:
                 entries += 1
+            else:
+                print('Invalid CVE has been detected.')
+                continue
 
             # Check if the leak has been published today.
-            if utils.midnight(datetime.today()) <= date:
+            if utils.midnight(datetime.today() - timedelta(days=7)) <= date:
```

Comment:
This is now using a time window of 7 days, but the `event_exists` function in `utils.py` is using `date.today())) - config.CHECK_INTERVAL_SECONDS * 2` until now, and the check interval is set to an hour.